tests/testsuite.src/run_fundamental.at

```diff
@@ -6178,3 +6178,41 @@ prog.cob:7: note: value size is 3
 AT_CHECK([$COBCRUN_DIRECT ./prog], [0], [], [])
 
 AT_CLEANUP
+
+AT_SETUP([autodetect format])
+AT_KEYWORDS([cobc free])
+
+AT_DATA([prog.cob], [
+  IDENTIFICATION   DIVISION.
+  PROGRAM-ID.      prog.
+  DATA             DIVISION.
+  WORKING-STORAGE  SECTION.
+  PROCEDURE        DIVISION.
+     DISPLAY "OK" NO ADVANCING
+     END-DISPLAY.
+     DISPLAY "KO" NO ADVANCING
+     END-DISPLAY.
+     STOP RUN.
+])
+
+AT_CHECK([$COMPILE -fformat=auto prog.cob], [0], [],
```

Comment:
Please adjust to 

```m4
AT_CHECK([$COMPILE prog.cob], [0], [],
[prog.cob:2: note: automatically switching to free format
])
```

-->  include source reference, setting the info as note, use lower-case

Please move the test to syn_...something, change to COMPILE_ONLY and drop the run.

One check missing (can be done at the end of this test): AT_DATA ([prog2.cob] which is fixed format, then compile both together (and have the message only for the first one.

Reply:
I created a file `syn_format.at`, as `syn_misc.at` seems to be already too long.
I added more examples, including with BOM (it seems it was not tested before).

Reply:
So, I moved the test back to `syn_misc.at` :-)

Reply:
... one more to add to https://sourceforge.net/p/gnucobol/feature-requests/search/?q=testsuite - I've just did that with https://sourceforge.net/p/gnucobol/feature-requests/430/.